src/test/java/software/amazon/cloudwatchlogs/emf/model/MetricDefinitionTest.java

```diff
@@ -31,6 +31,27 @@ public void testThrowExceptionIfNameIsNull() {
         new MetricDefinition(null);
     }
 
+    @Test
+    public void testSerializeMetricDefinitionWithoutUnitWithStorageResolution()
+            throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        MetricDefinition metricDefinition =
+                new MetricDefinition("Time", StorageResolution.HIGH, 10);
+        String metricString = objectMapper.writeValueAsString(metricDefinition);
+
+        assertEquals("{\"Name\":\"Time\",\"Unit\":\"None\",\"StorageResolution\":1}", metricString);
+    }
+
+    @Test
+    public void testSerializeMetricDefinitionWithUnitWithoutStorageResolution()
+            throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        MetricDefinition metricDefinition = new MetricDefinition("Time", Unit.MILLISECONDS, 10);
```

Comment:
Can we have an identical test that creates a MetricDefinition as such
```
new MetricDefinition("Time", StorageResolution.STANDARD, 10);
```

This way we can check that standard resolution is being excluded properly

Reply:
Added!